### ISSUE TITLE
CWA Backend Infrastructure Architecture document is outdated

### ISSUE THREAD

**Issue by ndegendogo** [corona-warn-app/cwa-documentation#496]
## Where to find the issue

Your document https://github.com/corona-warn-app/cwa-documentation/blob/master/backend-infrastructure-architecture.pdf (Version 1 from 15.6.2020) is outdated, and does not mention the European Federation Gateway. 
See especially 

1) Section 3.2.3.8, last sentence: 
"Note: please keep in mind, that the URLs and the implementation is still under development. Changes may occur and may not reflect the actual implementation.“
=> So maybe now 6 months after go-life it is time to update the document?

2) Section 4.3 seems outdated; the European Federation Gateway is nowhere mentioned in the whole document.

Furthermore I found the following minor issues and typos in the document:

3) Headlines are not consistently capitalized.
Examples: 2.1.2, 3.2.2, 3.2.3, 3.2.3.5, 3.2.4, 3.2.4.1, 3.2.5, 4.6, 4.6.2,4.6.4
Table 3

4) Corona Warn App is spelled inconsistently: „Corona Warn App“ vs „Corona-Warn-App“

5) Table of Tables has wrong numbering

6) Section 2.1. / Page 6: „New identifiers are derived from a Temporary Exposure Key (TEK) that is substituted at midnight (UTC) every day through means of cryptography“ 
=> wording is ambiguous (cryptography is used for key derivation, not for TEK substitution).
Suggestion: „New identifiers are derived through means of cryptography from a Temporary Exposure Key (TEK) that is substituted at midnight (UTC) every day“

7) Section 3.2.3.7: Copy-and-paste typo in last sentence? „For a complete …. of the OpenShift service ….“ => „Verification Portal“?

8) Section 3.2.3.8: Last sentence of first paragraph: typo? „content is then delivery“ => „content is then delivered“?

9) Section 3.2.5 / table 2 and table 3: no entries for DEK key distribution.
    table 2: last line: DNS Name has a blank - is this a typo?
    table 3: first line: Connection path: typo (Verifiaction-Server => Verification-Server)

---
Internal Tracking ID: EXPOSUREAPP-4491

**Comment by ndegendogo** [comment]
@heinezen is it true that you are currently trying to get this document updated? (see [this comment](https://github.com/corona-warn-app/cwa-documentation/issues/450#issuecomment-752129355) )

Could you then try to get also the abovementioned topics covered?

**Comment by heinezen** [comment]
@ndegendogo 

The original request for updates were only to update the server domains I think. Thereffore, I've mirrored your whole issue to Jira (ticket ID: EXPOSUREAPP-4491). The dev team will address this and the other outdated documentation soon.

Regards,
CH

----
Corona-Warn-App Open Source Team

**Comment by ndegendogo** [comment]
@heinezen thanks.

With regards to the "minor findings" I could alternatively offer to prepare a PR; however, this should be aligned with your other changes to avoid unnecessary merge conflicts.

**Comment by heinezen** [comment]
@ndegendogo The dev team might replace the PDF with a markdown file and split it up. So I'll suggest you wait and review the PR for this issue once it is ready :)

----
Corona-Warn-App Open Source Team

**Comment by ndegendogo** [comment]
@heinezen markdown sounds like a good idea from view of maintaining the document in future 😀👍

**Comment by MikeMcC399** [comment]
There have been more changes in the app functionality which impact the backend and that are not documented in the https://github.com/corona-warn-app/cwa-documentation/blob/master/backend-infrastructure-architecture.pdf

See also https://github.com/corona-warn-app/cwa-website/issues/952 regarding "FAQ: Firewall setting for Event-driven User Surveys (EDUS)".

**Comment by larswmh** [comment]
Closing this issue as the related internal ticket is set to `obsolete`.

Developer comment:

> Architecture documents were updated in the meanwhile

**Comment by MikeMcC399** [comment]
@larswmh 

>Closing this issue as the related internal ticket is set to completed.
>
>Developer comment:
>
>>Architecture documents were updated in the meanwhile

The document [backend-infrastructure-architecture.pdf](https://github.com/corona-warn-app/cwa-documentation/blob/main/backend-infrastructure-architecture.pdf) still shows Version 1 15.6.2020 on its title page.

The date 15.6.2020 is the date of the last commit on this document, exactly two years ago, so this document has **not** been updated.
